src/components/PageResultsNew/index.css

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Do we want this style file, of we can use already existing style file.
Up to this moment all styles were at single point.

Reply:
I'd prefer to keep the specific tests in here for now as they only apply to this page.